src/gfx/follow_eye.cpp

```diff
@@ -0,0 +1,44 @@
+#include <context.hpp>
+#include <gfx/follow_eye.hpp>
+#include <maths/maths.hpp>
+
+namespace misc {
+namespace {
+struct circle_params_t {
+	sf::Vector2f position{};
+	sf::Color fill = sf::Color::White;
+	sf::Color outline = sf::Color::Transparent;
+	f32 radius = 25.0f;
+	u32 points = 64;
+};
```

Comment:
Adding the `scale` (x,y) values would be useful, but I can also add it in a future PR